### PR TITLE
Fix - missing `messageTemplates` in `ExcludeMimeType` validator

### PR DESCRIPTION
**Pull request by webimpress**
Fix for bug reported in #90

### CODE REVIEW

File: test/File/ExcludeMimeTypeTest.php [hunk 1]
```diff
@@ -51,6 +51,10 @@ public function testBasic($options, $isValidParam, $expected)
         $validator = new ExcludeMimeType($options);
         $validator->enableHeaderCheck();
         $this->assertEquals($expected, $validator->isValid($isValidParam));
+        if (!$expected) {
```
Comment (Maks3w) [inline test/File/ExcludeMimeTypeTest.php]: Please instead add the expected getMessages return value on the data provider.

File: test/File/ExcludeMimeTypeTest.php [hunk 2]
```diff
@@ -44,13 +48,18 @@ public function basicBehaviorDataProvider()
...
      * @dataProvider basicBehaviorDataProvider
-     * @return void
+     *
+     * @param string|array $options
+     * @param array $isValidParam
+     * @param bool $expected
+     * @param array $messages
      */
-    public function testBasic($options, $isValidParam, $expected)
+    public function testBasic($options, array $isValidParam, $expected, array $messages = [])
```
Comment (Maks3w) [inline test/File/ExcludeMimeTypeTest.php]: It's better to have all possible values in the data provider for get the whole picture. Please remove the default value.

File: test/File/ExcludeMimeTypeTest.php [hunk 3]
```diff
@@ -25,17 +25,21 @@ public function basicBehaviorDataProvider()
...
-            //    Options, isValid Param, Expected value
-            ['image/gif',                      $fileUpload, true],
-            ['image',                          $fileUpload, false],
-            ['test/notype',                    $fileUpload, true],
-            ['image/gif, image/jpeg',          $fileUpload, false],
+            //    Options, isValid Param, Expected value, messages
+            ['image/gif',                 $fileUpload, true],
+            ['image',                     $fileUpload, false, [ExcludeMimeType::FALSE_TYPE => "File has an incorrect mimetype of 'image/jpeg'"]],
+            ['test/notype',               $fileUpload, true],
+            ['image/gif, image/jpeg',     $fileUpload, false, [ExcludeMimeType::FALSE_TYPE => "File has an incorrect mimetype of 'image/jpeg'"]],
```
Comment (Maks3w) [inline test/File/ExcludeMimeTypeTest.php]: You can avoid repeat the same message using a local variable `$falseTypeMessage`

Comment (Maks3w) [inline test/File/ExcludeMimeTypeTest.php]: https://github.com/zendframework/zend-inputfilter/blob/master/test/InputTest.php#L688

This is an example of a readable data provider. Notice even we mask the scalar values with a variable name so you can understand the meaning inmediatelly.

File: src/File/ExcludeMimeType.php [hunk 4]
```diff
@@ -22,6 +22,15 @@ class ExcludeMimeType extends MimeType
     const NOT_READABLE = 'fileExcludeMimeTypeNotReadable';
 
     /**
+     * @var array Error message templates
+     */
+    protected $messageTemplates = [
+        self::FALSE_TYPE   => "File has an incorrect mimetype of '%type%'",
```
Comment (Maks3w) [inline src/File/ExcludeMimeType.php]: I think INCORRECT_TYPE it's a better name. Why FALSE?

Comment (webimpress) [inline src/File/ExcludeMimeType.php]: Constant was defined, the same constants name are in parent class `MimeType`, I've added only messages.
We can change it, maybe it is even better, but for consistency I'd prefer to keep `FALSE_TYPE` ...
And it's not in the scope of the issue ;)

Comment (Maks3w) [inline src/File/ExcludeMimeType.php]: ok. You're right.

**Comment by weierophinney** [comment]
@Maks3w Please don't forget to update the CHANGELOG when merging... (I'll do it now, so I can release this bugfix.)

**Comment by Maks3w** [comment]
I was waiting for add more patches to the release

**Comment by weierophinney** [comment]
No need to wait on releases anymore! :-) My rule of thumb is that if I
won't complete reviewing patches in the same session, release immediately,
and do the other patches in the next release. This is also why it's
important to do change logs on merge; it ensures nobody has to look up
what's been merged when doing a release.

**Pull request merged**